cpu/esp_common/Makefile.include

```diff
@@ -202,26 +133,6 @@ else
   FFLAGS += 0x10000 $(FLASHFILE).bin
 endif
 
+# NOTE: These checks can be turned into normal conditionals when #9913 is fixed
 # increase the test timeout for file system tests that use the SPI flash drive
-
-ifneq (,$(filter spiffs,$(USEMODULE)))
-  export RIOT_TEST_TIMEOUT = 120
-endif
-
-ifneq (,$(filter littlefs,$(USEMODULE)))
-  export RIOT_TEST_TIMEOUT = 120
-endif
-
-# increase the number of netif for each enabled network interface
-
-ifneq (,$(filter esp_wifi,$(USEMODULE)))
-  $(eval GNRC_NETIF_NUMOF=$(shell echo $$(($(GNRC_NETIF_NUMOF)+1))))
-endif
-
-ifneq (,$(filter esp_now,$(USEMODULE)))
-  $(eval GNRC_NETIF_NUMOF=$(shell echo $$(($(GNRC_NETIF_NUMOF)+1))))
-endif
-
-ifneq (,$(filter esp_eth,$(USEMODULE)))
-  $(eval GNRC_NETIF_NUMOF=$(shell echo $$(($(GNRC_NETIF_NUMOF)+1))))
-endif
+export RIOT_TEST_TIMEOUT = $(if $(filter spiffs littlefs,$(USEMODULE)),120)
```

Comment:
I would not export this variable all the time but only with the `test` target (untested suggestion):
```suggestion
RIOT_TEST_TIMEOUT = $(if $(filter spiffs littlefs,$(USEMODULE)),120)
$(call target-export-variables,test,$(RIOT_TEST_TIMEOUT))
```

Reply:
Added a check in fdcf9fd

Reply:
Oh sorry, just saw the suggestion, will change it

Reply:
Done.